examples/elm.json

```diff
@@ -1,25 +1,25 @@
 {
     "type": "application",
     "source-directories": [
-        "."
+        "src"
```

Comment:
This should not have been changed (the elm files are in the root examples directory)